consultation_analyser/pipeline/ml_pipeline.py

```diff
@@ -1,20 +1,39 @@
 import logging
+import json
+
+from django.core.serializers.json import DjangoJSONEncoder
```

Comment:
Doesn't the JSON field handle serialization for us?

Reply:
Don't know - need to investigate more!